### PR TITLE
Organizing

### PR DESCRIPTION
**Pull request by mariosant**
This PR is mostly about organizing code into directories and seperating concepts.

### CODE REVIEW

File: .meteor/packages [hunk 1]
```diff
@@ -16,9 +16,9 @@ standard-minifiers      # JS/CSS minifiers run for production mode
 es5-shim                # ECMAScript 5 compatibility for older browsers.
 ecmascript              # Enable ECMAScript2015+ syntax in app code
 
-autopublish             # Publish all data to the clients (for prototyping)
```
Comment (mariosant) [inline .meteor/packages:19]: Removing autopublish, will make us think about published data from the start. In order for data to be accessible from the client, we will have to "publish" it at the server.

You can think of data "publications" as REST endpoints if you like. You ask endpoints to bring you data (by subscribing to them) and so they do.

File: client/main.html [hunk 2]
```diff
@@ -12,17 +12,4 @@
       ga('send', 'pageview');
 
     </script>
-    <div class="container">
```
Comment (mariosant) [inline client/main.html:15]: Iron router handles the active view by appending it to the `<body>` tag. It is currently set to display the `home` template at `/` route

File: routes/routes.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+Router.configure({
+  layoutTemplate: 'ApplicationLayout'
```
Comment (mariosant) [inline routes/routes.js:2]: Make default layout `ApplicationLayout`. This is always overridable.

**Comment by howardroark** [comment]
Fantastic! I'll just merge this right away.  Thanks!

File: server/main.js [hunk 4]
```diff
@@ -1,6 +0,0 @@
-Meteor.startup(function () {
```
Comment (mariosant) [inline server/main.js:1]: No need for that.

**Pull request merged**